html/index.html

```diff
@@ -1,115 +1,86 @@
+        <button type="button" class="bootstrap icon icon-directory btn btn-default btn-sm" onclick="DOM.promptNewDir()">New Dir</button>
+        <button type="button" class="bootstrap icon icon-upload btn btn-default btn-sm" onclick="CloudCmd.Upload.show();">Upload</button>
+        <label class="bootstrap checkbox-inline"><input type="checkbox" class="bootstrap" id="checkbox-dotfiles" onclick="dotfilesChecked()" value="">Show Dotfiles</label>
+        <label class="bootstrap checkbox-inline"><input type="checkbox" class="bootstrap" id="checkbox-ownermode" onclick="ownerModeChecked()" value="">Show Owner/Mode</label>
+    </div>
+    <div class="fm">{{ fm }}</div>
+    <div id="js-keyspanel" class="keyspanel">
+        <!-- OSC_CUSTOM_CODE remove this button >> <button id=f1      class="cmd-button reduce-text icon-help"      title="Help"            >F1</button> -->
+        <button id=f2      class="cmd-button reduce-text icon-rename"    title="Rename"          >F2</button>
+        <button id=f3      class="cmd-button reduce-text icon-view"      title="View"            >F3</button>
+        <button id=f4      class="cmd-button reduce-text icon-edit "     title="Edit"            >F4</button>
+        <button id=f5      class="cmd-button reduce-text icon-copy"      title="Copy"            >F5</button>
+        <button id=f6      class="cmd-button reduce-text icon-move"      title="Move"            >F6</button>
+        <button id=f7      class="cmd-button reduce-text icon-directory" title="New Directory"   >F7</button>
+        <button id=f8      class="cmd-button reduce-text icon-delete"    title="Delete"          >F8</button>
+        <button id=f9      class="cmd-button reduce-text icon-menu"      title="Menu"            >F9</button>
+        <button id=f10     class="cmd-button reduce-text icon-config"    title="Config"          >F10</button>
+        <!-- OSC_CUSTOM_CODE Convert console button to a link to wetty and remove the original defaults -->
+        <a onclick='window.open(CloudFunc.getWettyLink(CloudFunc.Path))' target="_blank"><button id=wetty       class="cmd-button reduce-text icon-console"   title="Wetty">~</button></a>
+        <!-- OSC_CUSTOM_CODE remove this button >> <button id=contact class="cmd-button reduce-text icon-contact"   title="Contact"         ></button> -->
+    </div>
+</body>
 
-<script src="{{ prefix }}/lib/client/ood_tree.js"></script>
+<script src="{{ prefix }}/cloudcmd.js"></script>
 <script>
+CloudCmd('{{ prefix }}');
+
+// Event handler activates on page load
 jQuery12('body').on('panel_initialized', function(e){
     initTree("{{ treeroot }}");
+    document.getElementById("checkbox-dotfiles").checked = ( getCookie('dotfiles') === 'true' );
+    dotfiles();
+    document.getElementById("checkbox-ownermode").checked = ( getCookie('ownermode') === 'true' );
+    ownerMode();
 });
```

Comment:
This should be before `CloudCmd('{{ prefix }}');` not after. In fact, the safest is before any of Cloud Commander's own code is pulled in:

```html
<script src="{{ prefix }}/lib/client/ood_tree.js"></script>
<script src="{{ prefix }}/lib/client/ood.js"></script>
<script>
// Event handler activates on page load
jQuery12('body').on('panel_initialized', function(e){
    initTree("{{ treeroot }}");
    document.getElementById("checkbox-dotfiles").checked = ( getCookie('dotfiles') === 'true' );
    dotfiles();
});
</script>


<script src="{{ prefix }}/cloudcmd.js"></script>
<script>
CloudCmd('{{ prefix }}');
</script>
```